mathml/presentation-markup/operators/mo-paint-lspace-rspace-ref.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>&lt;mo&gt; paint lspace rspace</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <h1>LTR case</h1>
+
+    <p>The test passes if the arrow has a leading space of 100px, which is as wide as the black block to the left,
+      and a trailing space of 200px, which is as wide as the black block to the right.</p>
+
+    <math>
+      <mspace width="100px" height="10px" depth="10px" mathbackground="black"></mspace>
```

Comment:
can you please use style="background: black" so that one does not need mathbackground support for this test?

Reply:
Done!